imagebuildah/stage_executor.go

```diff
@@ -1556,6 +1568,22 @@ func (s *StageExecutor) commit(ctx context.Context, createdBy string, emptyLayer
 		Manifest:              s.executor.manifest,
 		UnsetEnvs:             s.executor.unsetEnvs,
 	}
+	// generate build output
+	if s.executor.buildOutput != "" {
+		src, err := s.builder.MakeContainerImageRef(options)
+		if err != nil {
+			return "", nil, errors.Wrapf(err, "error computing layer digests and building metadata for container %q", s.builder.ContainerID)
+		}
+		rc, _, err := src.ExtractRootfs()
```

Comment:
Why is it OK to ignore `errChan`?

(Alternatively, I could see an argument that `errChan` should not exist, and instead `rc.Close()` should be blocking not that channel and handling the error, and that users of `rc` should always be obliged to check the return value of `rc.Close()`. That feels conceptually clearer but might just be a lot of extra complexity for no benefit.)

Reply:
handled.